zerver/lib/test_helpers.py

```diff
@@ -158,12 +158,12 @@ def wrapper_execute(self, action, sql, params=()):
 
     def cursor_execute(self, sql, params=()):
         # type: (TimeTrackingCursor, NonBinaryStr, Iterable[Any]) -> None
-        return wrapper_execute(self, super(TimeTrackingCursor, self).execute, sql, params)  # type: ignore # https://github.com/JukkaL/mypy/issues/1167
```

Comment:
There is an exception -- the `super`'s argument needs to be readded here somehow.

Reply:
Because `queries_captured` is not a class. Why?

Reply:
I think because it could probably be moved to be one, with some effort.  Not sure it's worth it though.